examples/nlp_seq/train.py

```diff
@@ -26,8 +26,9 @@
 from absl import logging
 from flax import jax_utils
 from flax import linen as nn
+from flax.training import train_state
 from flax.metrics import tensorboard
-from flax import optim
+import optax
```

Comment:
nit: consider sorting imports alphabetically

Reply:
Thanks. done.

Reply:
did you push the changes?

Reply:
I still see

```python
from flax.training import train_state
from flax.metrics import tensorboard
```

sorting alphabetically, it should instead be 
```python
from flax.metrics import tensorboard
from flax.training import train_state
```

Reply:
Reordered.